vajram/vajram-codegen/src/main/groovy/com/flipkart/krystal/vajram/VajramPlugin.groovy

```diff
@@ -60,7 +57,8 @@ class VajramPlugin implements Plugin<Project> {
                 VajramCodeGenFacade.codeGenVajramImpl(
                         project.sourceSets.main.java.srcDirs,
                         compiledMainDir,
-                        mainGeneratedSrcDir)
+                        mainGeneratedSrcDir,
+                        project.configurations.compileClasspath)
```

Comment:
Shouldn't we make the same change in testCodeGenVajramImpl as well for consistency?